lang/en/settings.php

```diff
@@ -1496,6 +1496,16 @@
                 'help' => 'Optional text to prepend to exported metric names',
             ],
         ],
+        'prometheus_metrics' => [
+            'enable' => [
+                'description' => 'Prometheus Metrics Endpoint',
+                'help' => 'Enable the /metrics endpoint for scraping',
+            ],
+            'port' => [
```

Comment:
I'm not sure this setting is very valuable.  All it does is say "your princess is in another castle".